os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/os/tizenrt/osif_tizenrt.c

```diff
@@ -459,7 +475,7 @@ bool osif_msg_queue_create(void **pp_handle, uint32_t msg_num, uint32_t msg_size
 
 	pmqd = mq_open((const char *) mq_name, O_RDWR | O_CREAT, 0666, &attr);
 	if (pmqd == (mqd_t) ERROR) {
-		printf("[osif_msg_queue_create] mq open fail\r\n");
+		dbg("mq open fail\r\n");

```

Comment:
I think it is better to print the errno when failure.

Reply:
All errno will be print when failure.